datacapture/src/androidTest/java/com/google/android/fhir/datacapture/test/views/QuestionnaireItemDialogMultiSelectViewHolderFactoryEspressoTest.kt

```diff
@@ -261,6 +262,7 @@ class QuestionnaireItemDialogMultiSelectViewHolderFactoryEspressoTest {
   }
 
   @Test
+  @Ignore // TODO https://github.com/google/android-fhir/issues/1482 FIXME
```

Comment:
@Ignore takes a parameter where you can add the link:
```suggestion
  @Ignore("https://github.com/google/android-fhir/issues/1482")
```